bin/ci/ubuntu/build-and-test.sh

```diff
@@ -15,29 +21,42 @@ echo "using APP: $APP"
 JOBS=${NUM_PROCESSORS:-2}
 JOBS=$((JOBS+1))
 
-if [[ ${BUILD:-scons} == "cmake" ]]; then
+if [[ ${BUILD:-cmake} == "cmake" ]]; then
```

Comment:
Could you make this same change in `bin/ci/ubuntu/install-dependencies.sh`? The line will look like
```
if [[ ${BUILD:-} == cmake ]]; then
```
I'm also wondering if it would be useful to use `:=` instead of `:-`.

Reply:
I've updated `install-dependencies.sh`, but just to clarify, this is not used by jenkins at all. If/when we eliminate the other CI systems, this might be dead code.